fs/write_json.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+
+interface WriteJsonOption {
+  spaces?: number | string;
+  replacer?: (number | string)[] | null | Replacer;
+}
+
+type Replacer = (key: string, value: any) => any;
```

Comment:
Should be declared before it is used.  Also will be failing linting due to explicit use of any.